Assets/Mirror/Tests/Runtime/SceneObjectSpawningTests.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Text.RegularExpressions;
+using NUnit.Framework;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace Mirror.Tests
+{
+    public class SceneObjectSpawningTests
+    {
+        const string ScenePath = "Assets/Mirror/Tests/Runtime/Scenes/SceneObjectSpawningTestsScene.unity";
```

Comment:
can't you drag the scene into the test's public field?
for example:
![image](https://user-images.githubusercontent.com/16416509/79071845-1d0a0000-7cde-11ea-8415-653ee7474be8.png)

Reply:
I don't think that is possible for tests. If we want this as something we can change in the inspector we could make a ScriptableObject that would hold the settings